README.md

```diff
@@ -129,13 +129,16 @@ Usage: docker volume create [<VOLUME_NAME>]
 <pre>
 $ docker run \
 --detach \
+--publish [HOST_PORT] : [CONTAINER_PORT] \
```

Comment:
Is white space in here okay?

```suggestion
--publish [HOST_PORT]:[CONTAINER_PORT] \
```

Reply:
I put those spaces in for readability but I think we should keep it as close to the actual usage like your suggestion. Thanks for the review!